core/services/chainlink/testdata/config-invalid.toml

```diff
@@ -83,6 +83,7 @@ MinIncomingConfirmations = 0
 
 [[EVM]]
 ChainID = '99'
+LogBroadcasterEnabled = false
```

Comment:
For this specific chain setting I set it to `false`, so the result correspondingly not include the WS URL missing error. Otherwise modification to config_test.go at line 1367 will be expected.

Reply:
This file is specifically meant to provoke errors via invalid configuration. We should flip this to true in order to have coverage on the additional validation.

Reply:
Okay sounds good. Remove the false assignment, as default value is `true`